File: 프로그래머스/0/181932. 코드 처리하기/코드 처리하기.js
function solution(code) {
    let mode = 0;
    let ret = '';
    code.split('').forEach((v,idx) => {
        console.log(mode)
        if(mode === 1) {
            if(v === '1') mode = 0;
            else    idx % 2 === 1 ? ret += v : null;
        } else {
            if(v === '1') mode = 1;
            else    idx % 2 === 0 ? ret += v : null;
        }
    })
    if(!ret) return 'EMPTY'
    else return ret
}

File: 프로그래머스/0/181932. 코드 처리하기/코드 처리하기.py
def solution(code):
    ans = ''
    mode = 0
    for i,c in enumerate(code):
        if mode == 0:
            if c == "1": mode = 1
            elif i%2 == 0: ans += c 
        else:
            if c == "1": mode = 0
            elif i%2 == 1: ans += c
    return ans if len(ans) != 0 else "EMPTY"